### PR TITLE
Adding audio and video codec preference

### PR DESCRIPTION
**Pull request by ptankTwilio**
### Summary
- Added `SettingsViewConntroller` to select audio and video codec while connecting to a Room
- QuickStart and CallKitQuickStarts updated
- Created a root level `Utils` folder. The `Utils` is shared by all projects.
- Moved Utils.swift to root level `Utils` folder

### CODE REVIEW

**Comment by ceaglest** [comment]
The sizing looks odd to me.

screen shot 2017-09-22 at 4 05 42 pm

File: Utils/SettingsTableViewController.swift [hunk 1]
```diff
@@ -26,7 +23,17 @@ class SettingsTableViewController: UITableViewController {
...
     }
-
+    
+    func getDesclaimerSize() -> CGSize {
+        let desclaimerString: NSString = SettingsTableViewController.desclaimerText as NSString
+        
+        return desclaimerString.boundingRect(with: CGSize(width: self.tableView.frame.width-20,
+                                                          height: CGFloat.greatestFiniteMagnitude),
+                                             options: NSStringDrawingOptions.usesLineFragmentOrigin,
+                                             attributes: [NSFontAttributeName: UIFont.preferredFont(forTextStyle: UIFontTextStyle.footnote)],
```
Comment (ceaglest) [inline Utils/SettingsTableViewController.swift]: Minor - You may want to assign the UIFont to a `let`.

File: Utils/Settings.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//
+//  Settings.swift
+//  VideoQuickStart
```
Comment (ceaglest) [inline Utils/Settings.swift]: This utils file doesn't belong to a particular project. Should we use VideoQuickStart or something else?

File: VideoCallKitQuickStart/ViewController.swift [hunk 3]
```diff
@@ -72,7 +72,8 @@ class ViewController: UIViewController {
     // MARK: UIViewController
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        self.title = "QuickStart"
```
Comment (ceaglest) [inline VideoCallKitQuickStart/ViewController.swift]: Should this say `CallKit QuickStart`?

Comment (ptankTwilio) [reply VideoCallKitQuickStart/ViewController.swift]: QuickStart is OK I think

Comment (ceaglest) [reply VideoCallKitQuickStart/ViewController.swift]: Either is fine with me. 👍

**Pull request merged**